src/parse.ts

```diff
@@ -110,6 +111,12 @@ const TOKEN_TO_REGEX: { [key: string]: RegExp } = {
     SSS: /(\d\d\d)/,
     A: /(AM|PM)/,
     a: /(am|pm)/,
+    MMMM: seqToRE([...MONTHS_EN]),
```

Comment:
issue: destructuring for object copying should be done inside the `seqToRE`. This makes it safer from potential errors that could arise if someone is not aware of the sorting performed in `seqToRE`.

Reply:
updated